Core/Src/main.c

```diff
@@ -19,6 +19,8 @@
 /* USER CODE END Header */
 /* Includes ------------------------------------------------------------------*/
 #include "main.h"
+#include "spi.h"
+#include "tim.h"
```

Comment:
Could you also push the rest of the files, including those files that are auto-generated by the cude ide? I could know the timer configuration by checking tim.c.